doc/fwutil/fwutil.md

```diff
@@ -252,35 +271,69 @@ Chassis1           BIOS        0ACLH004_02.02.007  Chassis BIOS
-          Module1  CPLD        <image_path>/cpld.bin  5 / 10                                   update is required
-                   FPGA        <image_path>/fpga.bin  5 / 5                                    up-to-date
+Chassis   Module   Component   Firmware               Version (current/available)              Status              Required Boot Action
+--------  -------  ----------  ---------------------  ---------------------------------------  ------------------  -----------------------
+Chassis1           BIOS        <image_path>/bios.bin  0ACLH004_02.02.007 / 0ACLH004_02.02.010  update is required  N/A
+                   CPLD        <image_path>/cpld.bin  5 / 10                                   update is required  N/A
+                   FPGA        <image_path>/fpga.bin  5 / 5                                    up-to-date          N/A
+                   SSD         <image_path>/ssd.bin   4 / 5                                    update is required  N/A
+          Module1  CPLD        <image_path>/cpld.bin  5 / 10                                   update is required  N/A
+                   FPGA        <image_path>/fpga.bin  5 / 5                                    up-to-date          N/A
+```
+
+4. Custom FW Package when the "boot" key and value are available in `platform_components.json`
+```bash
+root@sonic:~# fwutil show updates --fw-image=fw_update.tar.gz
+Component   Firmware               Version (current/available)              Status              Required Boot Action
+----------  ---------------------  ---------------------------------------  ------------------  -----------------------
+BIOS        <image_path>/bios.bin  0ACLH004_02.02.007 / NA                  update is required  None
+CPLD        <image_path>/cpld.bin  5 / 10                                   update is required  Power cycle
+FPGA        <image_path>/fpga.bin  5 / NA                                   up-to-date          Cold reboot
+SSD         <image_path>/ssd.bin   4 / 5                                    update is required  Cold reboot, Fast reboot
+```
+
+5. Auto-update status
+```bash
+root@sonic:~# fwutil show auto-updates status
+Component   Firmware               Version (current/available)              Status
+----------  ---------------------  ---------------------------------------  -------------------------
+CPLD        <image_path>/cpld.bin  5 / 10                                   updated but need a power-cycle
+SSD         <image_path>/ssd.bin   4 / 5                                    scheduled in cold reboot
 ```
 
 **Supported options:**
 1. -i|--image - show updates using current/next SONiC image
+2. -z|--fw-image - show updates using custom FW package
 
 **Note:** the default option is _--image=current_
 
 #### 2.2.2.2 Install commands
 
 ##### 2.2.2.2.1 Overview
 
-The purpose of the install commands group is to provide an interface  
```

Comment:
@sujinmkang why do we need to remove these whitespaces? Basically in .md two space notation is used to start a new line which allows text separation.

Reply:
reverted.